src/runtime/components/forms/SelectMenu.vue

```diff
@@ -423,21 +431,37 @@ export default defineComponent({
       return query.value === '' ? null : { [props.optionAttribute]: query.value }
     })
 
+    function clearOnClose () {
+      if (props.clearSearchOnClose) {
+        query.value = ''
+      }
+    }
+
+    function clearOnChange () {
```

Comment:
Why not call this `clearOnUpdate` for consistency?

Reply:
I deleted.